Tactility/Source/app/i2csettings/I2cSettings.cpp

```diff
@@ -64,7 +64,7 @@ static void show(lv_obj_t* parent, const hal::i2c::Configuration& configuration)
     // Frequency:
     if (configuration.config.mode == I2C_MODE_MASTER) {
         lv_obj_t* frequency_label = lv_label_create(card);
-        lv_label_set_text_fmt(frequency_label, "Frequency: %lu Hz", configuration.config.master.clk_speed);
+        lv_label_set_text_fmt(frequency_label, "Frequency: %u Hz", configuration.config.master.clk_speed);
```

Comment:
Regarding `clk_speed` and the voltage `uint32_t` values:
`lu` seems to be correct here, as `clk_speed` is a `uint32_t` and `long` is defined as "is at least 32 bits"?

I might be misunderstanding it, though.

Reply:
`uint32_t` is an `unsigned int` expanding. Therefore, it is not necessary to use a long unsigned int (`%lu`)

Reply:
`uint32_t` is considered `long unsigned int` by ESP-IDF (ESP32 is a 32 bit platform)
![image](https://github.com/user-attachments/assets/54435322-fe21-4c88-88d0-3bdb479974bb)

Reply:
I understand that the ESP32 SDK is different from the standard Linux compiler. For the ESP32 SDK, uint32_t is an unsigned long, whereas for the Linux compiler, it is an unsigned int. In this case, a condition could be required for the ESP32 platform and the Simulator. What do you think?

Reply:
I'd rather not have a condition in the logging, because then I'd have to do that everywhere. I might revisit the warnings at some point, but warnings at this level are not my priority right now.

(e.g. there's probably some hundred deprecation Mutex acquire/release warnings that are way more important.)